spec/features/projects/projects_index_spec.rb

```diff
@@ -202,7 +202,16 @@ def expect_projects_in_order(*projects)
       allow_enterprise_edition
     end
 
-    scenario 'CF columns and filters are visible' do
+    scenario 'CF columns and filters are not visible by default' do
+      load_and_open_filters admin
+
+      # CF's columns are not shown due to setting
+      expect(page).to_not have_text(custom_field.name.upcase)
+    end
+
+    scenario 'CF columns and filters are visible when added to settings' do
+      Setting.enabled_projects_columns << "cf_#{custom_field.id}"
+      Setting.enabled_projects_columns << "cf_#{invisible_custom_field.id}"
```

Comment:
Doing this breaks test isolation as the settings are carried over to the next test.